### ISSUE TITLE
Before we start

### ISSUE THREAD

**Issue by lstrzebinczyk** [lstrzebinczyk/home_management_system#1]
I have prepared a number of points to make before we begin.
1. I created a number of tickets for you, with as good descriptions as I can, and with **acceptance criteria**. Acceptance Criteria is a list of what must be implemented in each pull request to consider the ticket **Done**.
2. The process is going to look like this:
  - At the start of work on each ticket, we'll meet up and have a discussion about the ticket, make sure you understand everything that needs to be done.
  - You might decide the ticket is too large in scope and split the ticket in smaller ones.
  - You will create a new branch called "hms-(number-of-ticket-you-are-working-on)(-optional-additional-description)". For example `hms-2-signup-and-login`
  - You will work on the branch and implement the required behaviors
  - You will implement all of the ruby code, the visual side (html, css, js if needed), and write all of the required tests
  - I will be there for you to ask any additional technical questions if needed
  - Once daily we will meet for 15 minutes for you to give me a breakdown of your progress
  - Once you are done, I will review your PR. If I leave questions or ask you to change something, you will either answer or fix/change.
  - Note that if I ask about something, that is not automatically a silent ask to change it, or an indication that it's wrong. If it's a question, answer it. If I wish to have something done differentely, or think something can be done better, I will say so explicitly. 
  - I will also, likely, leave it for you to make a decision. That is also not a silent ask to do it my way : ).
  - Once I'm happy with your PR, I will approve it, you will be able to merge it, we'll close the ticket and we'll move on to the next one.
3. While working on a ticket, please focus on the issue at hand. If you have ideas for further improvements, or changes: open a new ticket, explain what you wish changed, or explain your idea, and we'll move on to them in next steps.
4. I am creating mockups for you. Be aware, that I am not a designer: these mockups are a more-or-less suggestion of what you should be going for. In "real" project you would be expected to implement them precisely, but here some leeway is allowed.
5. For visual side: you are to work in a mobile-first way, but desktop is also expected to have a good experience.
6. For this project, we are aiming at a simple backend-rendered app. Do not go too wild at frontend magic. There might be a time for that later.

AC:
  - For a good start, create a new application in rails 8. I want to see and approve a bare rails 8 app.
